### PR TITLE
Bugfix/spe 495 add missing evseid to commands

### PR DESCRIPTION
**Pull request by akoshterek**

### CODE REVIEW

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java [hunk 1]
```diff
@@ -4,66 +4,77 @@
...
+            validateLength(args, 2);
 
-            int connectorIdArgIndex = 0;
-            validateNumeric(connectorIdArgIndex, args.get(connectorIdArgIndex));
+            validateEvseAndConnector(args, EVSE_ID_INDEX, CONNECTOR_ID_INDEX);
         }
     },
 
     UNPLUG {
+        private static final int EVSE_ID_INDEX = 0;
```
Comment (aleksandr-nesterov) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java]: Please move these constants to ConsoleCommand class.

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java [hunk 2]
```diff
@@ -4,66 +4,77 @@
...
-            int authKeyArgIndex = 0;
-            int connectorIdArgIndex = 1;
-            validateIdentifierString(authKeyArgIndex, args.get(authKeyArgIndex));
-            validateNumeric(connectorIdArgIndex, args.get(connectorIdArgIndex));
+            validateIdentifierString(RFID_INDEX, args.get(RFID_INDEX));
+            validateNumeric(EVSE_ID_INDEX, args.get(EVSE_ID_INDEX));
+        }
+
+        private void validateIdentifierString(int index, String arg) {
+            boolean validTokenId = !arg.isEmpty() && arg.matches("([a-z]|[A-Z]|[0-9]|\\*|-|_|=|:|\\+|\\||@|\\.){0,36}");
```
Comment (aleksandr-nesterov) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java]: Consider moving regex to Pattern.compile private static final field.

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationState.java [hunk 3]
```diff
@@ -65,8 +65,8 @@ public boolean isCharging(Integer evseId) {
         return findEvse(evseId).isCharging();
     }
 
-    public CableStatus getCableStatus(int connectorId) {
-        return findConnector(connectorId).getCableStatus();
+    public CableStatus getCableStatus(int evseId, int connectorId) {
```
Comment (aleksandr-nesterov) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationState.java]: Consider removing this method. As you already have Evse instance fetched using evseId in the class which calls getCableStatus

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/actions/Plug.java [hunk 4]
```diff
@@ -30,11 +31,11 @@
     @Override
     public void perform(StationState stationState, StationMessageSender stationMessageSender) {
 
-        if (stationState.getCableStatus(connectorId) != CableStatus.UNPLUGGED) {
+        if (stationState.getCableStatus(evseId, connectorId) != CableStatus.UNPLUGGED) {
             throw new IllegalStateException(String.format("Connector is not available: %s", connectorId));
         }
 
-        Evse evse = stationState.findEvseByConnectorId(connectorId);
+        Evse evse = stationState.findEvse(evseId);
```
Comment (aleksandr-nesterov) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/actions/Plug.java]: Please move this code to the top. Such as:
```
Evse evse = stationState.findEvse(evseId);
if (evse.getCableStatus(connectorId) != CableStatus.UNPLUGGED) {
            throw new IllegalStateException(String.format("Connector is not available: %s", connectorId));
}
```

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/actions/Unplug.java [hunk 5]
```diff
@@ -27,19 +28,17 @@
     @Override
     public void perform(StationState stationState, StationMessageSender stationMessageSender) {
 
-        if (stationState.getCableStatus(connectorId) == CableStatus.LOCKED) {
+        if (stationState.getCableStatus(evseId, connectorId) == CableStatus.LOCKED) {
             throw new IllegalStateException("Unable to unplug locked connector: " + connectorId);
         }
 
-        Evse evse = stationState.findEvseByConnectorId(connectorId);
+        Evse evse = stationState.findEvse(evseId);
```
Comment (aleksandr-nesterov) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/actions/Unplug.java]: Please move this code to the top. Such as:
```
Evse evse = stationState.findEvse(evseId);
if (stationState.getCableStatus(evseId, connectorId) == CableStatus.LOCKED) {
            throw new IllegalStateException("Unable to unplug locked connector: " + connectorId);
        }
```

File: simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/StationMessageSenderTest.java [hunk 6]
```diff
@@ -51,9 +51,9 @@
     @Mock
     WebSocketClient webSocketClientMock;
 
-    StationMessageSender stationMessageSender;
+    private StationMessageSender stationMessageSender;
 
-    BlockingQueue<WebSocketClientInboxMessage> queue;
+    private BlockingQueue<WebSocketClientInboxMessage> queue;
```
Comment (aleksandr-nesterov) [inline simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/StationMessageSenderTest.java]: Why do you use private here? I deliberately dont use any encapsulation in the unit-tests as its just noise. Junit5 does not require encapsulation as Junit4

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java (hunk 1 above)
Comment (akoshterek) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java]: It's different for AUTH command so it wouldn't work

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java (hunk 2 above)
Comment (akoshterek) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/cli/ConsoleCommand.java]: done

**Pull request merged**